edrumulus.ino

```diff
@@ -26,6 +26,7 @@ MIDI_CREATE_DEFAULT_INSTANCE();
 #  define MYMIDI                     MIDI
 #  define MIDI_CONTROL_CHANGE_TYPE   midi::ControlChange
 #  define MIDI_SEND_AFTER_TOUCH      sendAfterTouch
+#  define MIDI_SERIAL                38400
```

Comment:
I have chosen this default because Hairless MIDI also uses this as the default. For Teensy, I think that this value has no effect anyway. So, I think the original code can stay.

Reply:
Without the change, the teensy board is needlessly set to 38400 (when midi is enabled, and why wouldn't it be enabled?).  This slows doen the serial plotting.

Reply:
Ok, agreed.